aosp_diff/preliminary/external/drm_hwcomposer/0002-Fix-issue-have-OVERLAY-crtc-count-neq-connector-count.patch

```diff
@@ -0,0 +1,212 @@
+From ba5bcbbcf8a09eaf84c95e86f635846f10eb2abf Mon Sep 17 00:00:00 2001
+From: "Li, HaihongX" <haihongx.li@intel.com>
+Date: Mon, 2 Aug 2021 10:00:08 +0800
+Subject: [PATCH] Fix issue while crtc count not equal to connector count,
+ meanwhile there is overlay plane
+
+---
+ DrmHwcTwo.cpp           | 11 +++++++--
+ drm/DrmConnector.cpp    |  1 +
+ drm/DrmDevice.cpp       | 53 +++++++++++++++++++++++++++++++++++++++--
+ include/drmhwcomposer.h | 11 +++++++++
+ utils/hwcutils.cpp      | 16 +++++++++++++
+ 5 files changed, 88 insertions(+), 4 deletions(-)
+
+diff --git a/DrmHwcTwo.cpp b/DrmHwcTwo.cpp
+index db5f351..7178375 100644
+--- a/DrmHwcTwo.cpp
++++ b/DrmHwcTwo.cpp
+@@ -921,7 +921,7 @@ HWC2::Error DrmHwcTwo::HwcDisplay::GetDisplayVsyncPeriod(
+ }
+ 
+ HWC2::Error DrmHwcTwo::HwcDisplay::SetActiveConfigWithConstraints(
```

Comment:
please double confirm SetActiveConfigWithConstraints is necessary. Confused on that because virtio-gpu can work without problem.